src/relaycard/card.py

```diff
@@ -135,37 +138,26 @@ def setup(self) -> bool:
         return self.is_initialized
 
     def get_ports(self, address: int) -> RelayState:
-        response = self._execute_retry(CommandCodes.GETPORT, lambda r: r.command == ResponseCodes.GETPORT, address)
+        response = self._execute_retry(ComCodes.GETPORT, address)
         return RelayState(response.data)
 
     def get_port(self, address: int, port: int) -> bool:
-        if not (0 <= port <= 7):
```

Comment:
Why don't we validate the given port number anymore?

Reply:
It is done by `RelayState.get_port`. So it was duplicated